integrations/wootric/lib/index.js

```diff
@@ -96,23 +99,14 @@ Wootric.prototype.track = function(track) {
  * @param {Page} page
  */
 
-Wootric.prototype.page = function(page) {
+Wootric.prototype.page = function() {
   // Only track page if we haven't already tracked it
   if (this.lastPageTracked === window.location) {
     return;
   }
 
   // Set this page as the last page tracked
   this.lastPageTracked = window.location;
-
-  var wootricSettings = window.wootricSettings;
```

Comment:
We are 100% removing the tracking feature from `page()` right? do we need to update any doc on Segment's side to reflect this?

Reply:
Good question. Let me check if we mention that anywhere.

Reply:
Any update here?

Reply:
No update needed: https://segment.com/docs/destinations/wootric/#page

Reply:
What do you mean? it says `When you call page in the Javascript API, Wootric will log the page visit.` we won't be doing that anymore. I believe we need to remove, that don't we?

Reply:
You're right. My bad, we would need to ask Segment to update those docs.

Reply:
Maybe we should remove the `page()` method as a whole from the integration? Now that I look closer to the code, we won't be really using this, and we shouldn't require customers to call this method to be able to use the integration. What do you think?

Reply:
Yes, I think it's safe to remove it